packages/core/jkl/_typography.scss

```
@@ -217,3 +217,50 @@ $_text-styles: (
     font-weight: $typography-weight-bold;
     letter-spacing: -0.014em;
 }
+
+:root {
+    @each $level, $values in $_text-styles {
+        $variants: map.get($_text-styles, $level);
+        @each $property, $value in map.get($variants, "small-screen") {
+            --jkl-#{$level}-#{$property}: #{$value};
+        }
+    }
+
+    @include screens.from-medium-device {
+        @each $level, $values in $_text-styles {
+            $variants: map.get($_text-styles, $level);
+            @each $property, $value in map.get($variants, "large-screen") {
+                --jkl-#{$level}-#{$property}: #{$value};
+            }
+        }
+    }
+}
```

Disse kan forenkles bittelitt ved å fjerne `$variants` som mellomsteg (sass:map.get støtter å hente flere nivåer ned i map-et):

```suggestion
:root {
    @each $level, $values in $_text-styles {
        @each $property, $value in map.get($_text-styles, $level, "small-screen") {
            --jkl-#{$level}-#{$property}: #{$value};
        }
    }

    @include screens.from-medium-device {
        @each $level, $values in $_text-styles {
            @each $property, $value in map.get($_text-styles, $level, "large-screen") {
                --jkl-#{$level}-#{$property}: #{$value};
            }
        }
    }
}
```